web/docs/faqs.md

```diff
@@ -0,0 +1,204 @@
+# FAQs
+
+This page provides answers to frequently asked questions (FAQs) about Tenzir.
+
+## What is Tenzir?
+
+Tenzir is a platform that pioneers open source security data pipelines.
+
+Tenzir is also the name of the Germany-based startup behind the product.
+
+## What part of Tenzir is open and what part is closed source?
+
+The diagram below illustrates the high-level components, indicating what parts
+are open and what are closed source:
+
+![Open source vs. closed source](open-vs-closed-source.excalidraw.svg)
+
+The overwhelming majority of code is open source and available at under a
+permissive BSD 3-clause licence, available at
+[GitHub](https://github.com/tenzir/tenzir). This code implements the `tenzir`
+and `tenzir-node` [command line tools](command-line.md) that execute pipelines
+and store data. A flexible [plugin infrastructure](architecture/plugins.md)
+allows for enhancing the open-source core with closed-source plugins.
+
+We offer the [Community Edition](https://tenzir.com/pricing) as a binary package
+that consists of all open-source plugins, plus our own additional closed source
+plugins.
+
+The Tenzir stack also consists of a cloud platform and web app, accessible via
+[app.tenzir.com](https://app.tenzir.com). Both of these components are not
+openly available. The platform is the *control plane* that manages the nodes and
+pipelines, where as the nodes are the *data plane* where the data.
+
+## Can Tenzir see my data?
+
+In theory *yes*, in practice *no*. Let us explain.
+
+A Tenzir deployment consists of *nodes* that you manage, and a *platform* that
+we manage. The *app* runs in your browser. All computation and storage takes
+place only at your nodes. The platform acts as rendezvous point that connects
+two TLS-encrypted channels, one from the node to the platform, and one from the
+browser to the platform:
+
+![Platform Connections](platform-connections.excalidraw.svg)
+
+To date, we connect these two channels at the platform. This means we *could*
+interpose on data that travels from the node to the app, for example, when
+interacting with the [Explorer](setup-guides/use-the-app/README.md#explorer). However,
+data privacy is of utmost importance to our users. We are mission-driven company
+with strong ethics. Our engineering follows state-of-the-art
+infrastructure-as-code practices and we are performing security audits to ensure
+that our code quality meets the highest standards.
+
+We have plans to make this a single, end-to-end encrypted channel, so that we
+no longer have the theoretical ability to interpose on the data transfer between
+app and node.
+
+## Does Tenzir run on-premise?
+
+Our current version of the platform runs in the public cloud (AWS in Europe),
```

Comment:
```suggestion
Our current version of the platform runs in the public cloud (AWS in US),
```

@rdettai or @lava to confirm.

Reply:
no it's still europe, what you saw earlier was just to run the POC along the existing infra

Reply:
Okay, then I'll just resolve this. Thanks.